system-tests/resources/cli-tools/entrypoint.sh

```diff
@@ -16,11 +16,16 @@ for participant in "${PARTICIPANTS[@]}"; do
     echo "Registering $participantName"
     java -jar registration-service-cli.jar -d="$did" -k=/resources/vault/$participantName/private-key.pem -s='http://registration-service:8184/authority' participants add --ids-url "http://$participantName:8282"
 
-    echo "Seeding VC for $participantName"
-    vcId=$(uuidgen)
-    java -jar identity-hub-cli.jar -s="http://$participantName:8181/api/identity-hub" vc add \
-                -c='{"id":"'$vcId'","credentialSubject":{"region":"'$region'"}}' \
-                -b="$did" \
-                -i="did:web:did-server:gaia-x" \
-                -k="/resources/vault/gaia-x/private-key.pem"
+   for subject in '"region": "'$region'"' '"gaiaXMember": "true"'
```

Comment:
nit: do you prefer this `for` here? because on the first read it gave me the impression that it is to loop over something, but I think that is not the intention here.

If sounds okay to you then we can do this without `for` and directly keep jsonkeys at the place of $subject.

Reply:
We do loop
- once with  $subject = '"region": "'$region'"'
- once with  $subject = '"gaiaXMember": "true"'

Reply:
Yeah I see now. Thanks